includes/Plugin.php

```diff
@@ -214,33 +193,4 @@ protected function get_delegations(): array {
 			},
 		];
 	}
-
-	/**
```

Comment:
This is for backward compatibility, why was this removed?

Reply:
Don't need to keep BC stuff forever.

Plus, I couldn't find a single plugin or theme using any of these.